### ISSUE TITLE
Notifications are duplicated 

### ISSUE THREAD

**Issue by FernandoMiguel** [fetlife/android#567]
The latest release of the app has a new bug.
All notifications show up duplicate in the notification list inside app, and trying to open from the notification tray just opens the app from where it last was, not to where it should go

### Environment

 - Device: pixel 3xl
 - OS version: android Q Beta 3
 - App version. 302

**Comment by kistitan** [comment]
Hey @FernandoMiguel ,

Thank you for reporting this issue.
This is very unfortunate as you know, we are closing Android support today.

We quickly tried to reproduce the issue on our test devices, but so far everything works as expected.
Also 3.0.2 and 3.0.2-beta.3 are identical releases, so we did not really introduce any change here...

I know this is one of the less supportive replies a user usually get, but would you mind as a first step trying to completely reinstall the App?

If that does not help, I will try to see if there is anything I can find today, but to be honest, if we cannot see the issue on larger scale, probably we won't release a new version, to avoid introducing larger scale issues in the very last minute.

If the issue still persist, what I can still try to help you, as from next week fork the current repository and try to solve the issue for you specifically, however in that case I am afraid you won't have the official FetLife app anymore.

Dreamlite

**Comment by FernandoMiguel** [comment]
yeah man, well aware of the precarious future for you and the development.
I'll try to clear data and reinstall

**Comment by kistitan** [comment]
@FernandoMiguel 

Hey, found reason for the duplication.
It should be solved in 3.0.3!

Thank you

**Comment by FernandoMiguel** [comment]
Great.
Now just need to work around mfa before I can login again lol
